x-pack/plugins/security_solution/common/endpoint/data_generators/exceptions_list_item_generator.ts

```diff
@@ -73,6 +64,19 @@ export class ExceptionsListItemGenerator extends BaseDataGenerator<ExceptionList
       updated_by: this.randomUser(),
       ...(overrides || {}),
     };
+
+    // If the `entries` was not overwritten, then add in the PATH condition with a
+    // value that is OS appropriate
+    if (!overrides.entries) {
```

Comment:
I noticed the prior implementation was not actually looking at the OS type that was being generated. It will now always look at it, as long as the `entries` was not overwritten by the caller

Reply:
🔥